packages/cycle-scripts/template/config/javascript/flavor.js

```diff
@@ -0,0 +1,24 @@
+const dependencies = []
```

Comment:
Also can we simply use `template/src/<template>` instead of having a whole seperate structure just for those files? as they are strictly related to the templates. Perhaps we could renane this file into `templateStrings.js` ?